components/raftstore/src/store/config.rs

```diff
@@ -546,6 +555,10 @@ impl Config {
             ));
         }
 
+        if self.check_leader_lease_interval.as_millis() == 0 {
+            self.check_leader_lease_interval = self.raft_base_tick_interval;
```

Comment:
How about setting it to `self.raft_store_max_leader_lease - raft_base_tick_interval`?

Reply:
The tick tries to renew lease at about `self.raft_store_max_leader_lease / 4` in advance.
If we set it to `self.raft_store_max_leader_lease - raft_base_tick_interval` then the max period with no lease could be `self.raft_store_max_leader_lease - raft_base_tick_interval - self.raft_store_max_leader_lease / 4`

Reply:
Maybe `self.raft_store_max_leader_lease / 4` is a good choice.